README.md

```diff
@@ -1 +0,0 @@
-# dynamodb-source
```

Comment:
As discussed, if the README.rst file is present, then it will be reflected. Hence, not adding the duplicated README.md file.